documentation/modules/proc-configuring-kafka-bridge-tracing.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-config.adoc
+
+[id='proc-configuring-kafka-bridge-tracing-{context}']
+= Configuring distributed tracing
+
+[role="_abstract"]
+Enable distributed tracing to trace messages consumed and produced by the Kafka Bridge, and HTTP requests from client applications.
+
+Properties to enable tracing are present in the `application.properties` file.
+To enable distributed tracing, do the following:
+
+* Set the `bridge.tracing` property value to enable the tracing you want to use. Possible values are `jaeger` and `opentelemetry`.
+* Set environment variables for tracing.
+
+With the default configuration, you can implement a Jaeger tracing system based on OpenTelemetry or OpenTracing.
+
+OpenTelemetry and OpenTracing are API specifications for collecting tracing data as _spans_ of metrics data.
+Spans represent a specific operation.
+A trace is a collection of one or more spans.
+
+Traces are generated when the Kafka Bridge does the following:
+
+* Sends messages to HTTP clients
+* Consumes messages from HTTP clients
```

Comment:
Still not sure .... what if something like this?

"Sends message back from Apache Kafka to HTTP clients when they are consuming"
"Get messages from HTTP clients to send to Apache Kafka when they are producing"

Reply:
Trying to make it more succinct:

* Sends messages from Kafka to consumer HTTP clients 
* Receives messages from producer HTTP clients to send to Kafka